streamflow/cwl/translator.py

```diff
@@ -2062,6 +2109,13 @@ def _translate_workflow_step(
                     combinator_step.add_input_port(port_name, input_ports[global_name])
                     input_ports[global_name] = workflow.create_port()
                     combinator_step.add_output_port(port_name, input_ports[global_name])
+                if scatter_transformer:
+                    for global_name in scatter_inputs:
+                        port_name = posixpath.relpath(global_name, step_name)
+                        scatter_transformer.add_input_port(port_name, size_port)
+                    size_port = workflow.create_port()
+                    scatter_transformer.add_output_port("__size__", size_port)
```

Comment:
If size transformers do just this, move creation of `__size__` port directly in the constructor